### ISSUE TITLE
[a11y]: The ARIA attribute "aria-autocomplete" is in conflict with the corresponding HTML attribute "autocomplete"

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#17181]
[Reported in slack](https://ibm-studios.slack.com/archives/C2K6RFJ1G/p1723742794192229), this violation appears on downshift-based components with the latest Aug06 ruleset.

### Details

These attributes are set by downshift, the library that powers the Combobox, MultiSelect, FilterableMultiSelect and Dropdown.

While the spec calls for native attributes to "win" and be used if both of these are applied, in practice both are required in attempt to achieve the outcome of:

1. [Turning off form autocompletion](https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Turning_off_form_autocompletion) so the browser does not display it's own listbox that conflicts with the listbox provided by the component (see examples https://github.com/carbon-design-system/carbon/issues/8993, and https://github.com/carbon-design-system/carbon/issues/12602)
2. AT's surfacing the presence of a `list` connected to these components.

The attributes are not semantically equivalent and [have disparate features](https://w3c.github.io/html-aam/#att-autocomplete-form)

### Current progress

* We're working with IBMa to determine if this is a valid violation or not
  * https://github.com/IBMa/equal-access/issues/2052
* We've asked for more clarification on this topic from the aria working group
  * https://github.com/w3c/aria/issues/2337

### Reproduction

https://stackblitz.com/edit/github-k5cekb?file=src%2FApp.jsx

### Components impacted

- ComboBox
- MultiSelect

**Comment by philljenkins** [comment]
See [comment on W3C issue](https://github.com/w3c/aria/issues/2337#issuecomment-2397445552)

**Comment by tay1orjones** [comment]
Thanks @philljenkins! Do you know when the next release of the checker will be?
